src/test_l1_dsFPD.c

```diff
@@ -1198,38 +1233,30 @@ void test_l1_dsFPD_positive_dsSetFPColor (void)
     UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 02: Set all valid indicators to dsFPD_STATE_ON using dsSetFPState()
-    // Step 03: Call dsSetFPColor() with all valid eIndicator and all valid eColor parameters from yaml
     count = UT_KVP_PROFILE_GET_UINT8("dsFPD/Number_of_Indicators");
```

Comment:
Number_of_Indicators: 1. Please add this in Source_FPD.yaml. It is missing and it will cause parsing problem

Reply:
Added to source YAML